src/main/java/pl/net/rogala/eventy/service/EventService.java

```diff
@@ -2,23 +2,64 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.Sort;
+import org.springframework.security.core.Authentication;
 import org.springframework.stereotype.Service;
 import pl.net.rogala.eventy.entity.Event;
+import pl.net.rogala.eventy.entity.User;
+import pl.net.rogala.eventy.form.NewEventForm;
 import pl.net.rogala.eventy.repository.EventRepository;
+import pl.net.rogala.eventy.repository.UserRepository;
 
+import java.time.LocalDate;
 import java.util.List;
 
 @Service
 public class EventService {
 
     private EventRepository eventRepository;
+    private UserRepository userRepository;
+
+    private UserService userService;
+
+    private NewEventForm eventForm;
```

Comment:
To do wywalenia.

Reply:
done

Reply:
To komentarz do linijki 24 czyli `private NewEventForm eventForm;` ;-)

Reply:
done